infra-cdk/lib/stacks/ReportsBucketStack.ts

```diff
@@ -22,6 +22,13 @@ export class ReportsBucketStack extends cdk.Stack {
             bucketName: this.getBucketName(),
             blockPublicAccess: s3.BlockPublicAccess.BLOCK_ALL,
             removalPolicy: cdk.RemovalPolicy.RETAIN,
+            cors: [ //CORS configuration added
```

Comment:
Nit: the comment `//CORS configuration added` may not be necessary since that is pretty clear

If you would like a comment here, you can describe what CORS is or why you configured it this way :)